spec/deqm_test_kit/data_requirements_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+RSpec.describe DEQMTestKit::DataRequirements do
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('deqm_test_suite') }
+  let(:group) { suite.groups[3] }
+  let(:test_session) { repo_create(:test_session, test_suite_id: 'deqm_test_suite') }
+  let(:url) { 'http://example.com/fhir' }
+  # ensure this url matches url in embedded_client in data_requirements.rb
+  let(:embedded_client) do
+    'http://cqf_ruler:8080/cqf-ruler-r4/fhir'
+  end
+  let(:error_outcome) { FHIR::OperationOutcome.new(issue: [{ severity: 'error' }]) }
+
+  def run(runnable, inputs = {})
+    test_run_params = { test_session_id: test_session.id }.merge(runnable.reference_hash)
+    test_run = Inferno::Repositories::TestRuns.new.create(test_run_params)
+    Inferno::TestRunner.new(test_session: test_session, test_run: test_run).run(runnable, inputs)
+    Inferno::Repositories::TestRuns.new.results_for_test_run(test_run.id)
+  end
+
+  describe 'data requirements test' do
+    let(:test) { group.tests.first }
+    let(:measure_name) { 'EXM130' }
+    let(:measure_version) { '7.3.000' }
+    let(:test_id) { 'test_id' }
+
+    it 'passes if a Library was received' do
+      test_measure_response = FHIR::Bundle.new(total: 1, entry: [{ resource: { id: test_id } }])
+      test_library_response = FHIR::Library.new(dataRequirement: [{ type: 'Condition' }])
+
+      stub_request(:get, "#{url}/Measure?name=#{measure_name}&version=#{measure_version}")
```

Comment:
Since some of these stubs remain the same for every `it` check, can we pull them out under the `describe` umbrella so that they can be used across each check's context?

Reply:
This has proven a little bit tricky to address atm, so we can look into sharing `stub_request` s later